components/atoms/PullRequestOverviewChart/pull-request-overview-chart.tsx

```diff
@@ -58,40 +63,40 @@ const PullRequestOverviewChart: React.FC<PullRequestOverviewChartProps> = ({
   return (
     <div className="w-full h-1.5 flex gap-0.5 bg-light-slate-2 rounded-full overflow-hidden">
       {/* Open */}
-      {open && open > 0 && (
+      {open && open > 0 ? (
         <PullRequestOverviewChartBar
           onMouseOver={() => setOverviewDetails({ type: "open", percent: open })}
           percent={getPercentage(open)}
           type="open"
         />
-      )}
+      ) : null}
 
       {/* Merged */}
-      {merged && merged > 0 && (
+      {merged && merged > 0 ? (
         <PullRequestOverviewChartBar
           onMouseOver={() => setOverviewDetails({ type: "merged", percent: merged })}
           percent={getPercentage(merged)}
           type="merged"
         />
-      )}
+      ) : null}
```

Comment:
This was the actual fix... for some reasons, the `&&` condition renders a `"0"` in the dom for all `0%` type.

Reply:
It's because `draft` is `0` which is falsy, so it stops evaluating the rest and you end up with a `0`.